src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -232,6 +233,25 @@ Napi::Value CoreWrap::get_available_devices(const Napi::CallbackInfo& info) {
     return js_devices;
 }
 
+Napi::Value CoreWrap::get_versions(const Napi::CallbackInfo& info) {
+    const auto& devices_map = _core.get_versions(info[0].As<Napi::String>().Utf8Value());
```

Comment:
What if not string type would be passed or args would absent?

Reply:
I have added the required checks for this.